crates/aptos/src/node/analyze/analyze_validators.rs

```diff
@@ -334,16 +331,6 @@ impl AnalyzeValidators {
         let total_failures: u32 = failures.values().sum();
         let total_transactions: u32 = transactions.values().sum();
         let total_rounds = total_successes + total_failures;
-        assert_eq!(
```

Comment:
let's not change anything in the fetching/processing of the data, because it can get inconsistent (i.e. removal of this check), and instead filter on printing?


it only makes sense for print_detailed_epoch_table function, and so you can add it just there instead.

Reply:
Updated